700-quarkus-infinispan-grpc-kafka/src/main/proto/helloworld.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
```

Comment:
Is this .proto file included intentionally?

Reply:
Yes, this is to test the gRPC service.

Reply:
please make a note in readme about this file, not everybody is aware of protobuf stuff / have deep grpc insight

Reply:
Added